src/services/db/__tests__/RepoService.spec.ts

```diff
@@ -198,10 +207,10 @@ describe("RepoService", () => {
       })
 
       expect(actual).toEqual(expected)
-      expect(gitHubServiceCreate).toHaveBeenCalledWith(sessionData, {
-        content: mockContent,
-        fileName: mockFileName,
-        directoryName: mockDirectoryName,
+      expect(MockCommitServiceGitHub.create).toHaveBeenCalledWith(sessionData, {
+        content: "content",
+        fileName: "test.md",
+        directoryName: "",
```

Comment:
We should use `mockContent`, `mockFileName` and `mockDirectoryName` above, the actual call should also be updated too.

Reply:
b7fa30138bb1b5bacb5ff10b367e6c6d0d81473e

Reply:
b7fa30138bb1b5bacb5ff10b367e6c6d0d81473e